### ISSUE TITLE
where do you use the test_lables.csv file?

### ISSUE THREAD

**Issue by rivkaCode** [pankymathur/Fine-Grained-Clothing-Classification#3]
thank you for the gr8 work!
I saw that you used the train_lables.csv to train the model, i was wondering where you use the test_lables.csv file.
Thank you!

**Comment by pankajarm** [comment]
sorry for super delayed reply, I am not actively maintaining this repo, but ot answer your questions, correct, I haven't shown how to use test_labels.csv in current notebook. You can use test_labels.csv by following this approached by fastai v2, https://docs.fast.ai/tutorial.pets.html#Adding-a-test-dataloader-for-inference

The above documentation shows even how to create your custom test dataset by creating test dataloader, just make sure you are applying same transformation and normalization on test images the approached you used for training data set
